packages/bootstrap-vue-next/src/components/BTabs/BTab.vue

```diff
@@ -69,10 +69,10 @@ const parentData = inject(tabsInjectionKey, null)
 const computedId = useId(() => props.id, 'tabpane')
 const buttonId = useId(() => props.buttonId, 'tab')
 
-const activeBoolean = useBooleanish(() => props.active)
-const disabledBoolean = useBooleanish(() => props.disabled)
-const lazyBoolean = useBooleanish(() => props.lazyOnce ?? props.lazy)
-const noBodyBoolean = useBooleanish(() => props.noBody)
+const activeBoolean = computed(() => props.active)
+const disabledBoolean = computed(() => props.disabled)
+const lazyBoolean = computed(() => props.lazyOnce ?? props.lazy)
```

Comment:
```suggestion
const computedLazy = computed(() => props.lazyOnce ?? props.lazy)
```

Actual computed logic values like this can stay. But their variable name should probably be different.

Reply:
Generally, I agree.
But in this case, `computedLazy` clashes with an existing `computedLazy` (that does something different): https://github.com/bootstrap-vue-next/bootstrap-vue-next/pull/1791/files#diff-91a4211a7fd67ca96a56c6a3a098e92d7568ffad1258105546779bc809b90ff7R114

Do you have another preferred name?